doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,468 @@
+================================================
+Tutorial: Linear algebra on n-dimensional arrays
+================================================
+
+.. currentmodule:: numpy
+
+.. testsetup::
+
+   import numpy as np
+   np.random.seed(1)
+
+Prerequisites
+=============
+
+Before reading this tutorial you should know a bit of Python. If you
+would like to refresh your memory, take a look at the `Python
+tutorial <https://docs.python.org/tutorial/>`__.
+
+If you wish to work the examples in this tutorial, you must also have
+some software installed on your computer. Please see
```

Comment:
"some software" is a bit too general, we can spell it out (numpy, scipy, matplotlib).

I was a little on the fence about this, because we've always assumed such things. But on reflection I think this is quite nice; every tutorial can then really be standalone.